src/components/AppsSection.tsx

```diff
@@ -0,0 +1,37 @@
+import "./appsSection.scss";
+import Apps from "../res/Apps.svg";
+import Noti from "../res/Noti.svg";
+import Bolt from "../res/Bolt.svg";
+import Back from "../res/Back.svg";
+
+function AppsSection() {
+  return (
+    <div className="section apps" id="apps">
+      <h1 className="apps-header">Apps</h1>
+
+      <img alt="Apps" className="apps" src={Apps} />
```

Comment:
Reuse of class `apps`, we already use it for the section div. Maybe change to `apps-img`?